ru.json

```
@@ -0,0 +1,181 @@
+{
+	"footer": {
+		"description": "Modrinth — программное обеспечение с открытым кодом.",
+		"copyright": "© Guavy LLC",
+		"legal": {
+			"heading": "Legal",
+			"terms": "Условия использования",
+			"privacy": "Политика конфиденацильности",
+			"rules": "Правила"
+		},
+		"resources": {
+			"heading": "Ресурсы",
+			"blog": "Блог",
+			"docs": "Документация"
+		},
+		"buttons": {
+			"privacy": "Настройки конфиденциальности",
+			"theme": {
+				"label": "Изменить тему",
+				"system": "Системная тема",
+				"light": "Светлая тема",
+				"dark": "Тёмная тема",
+				"oled": "OLED тема"
+			},
+			"language": {
+				"label": "Выберите язык",
+				"en": "Английский",
+				"en-GB": "Английский (Соединённое Королевство)",
+				"ru-RU": "Русский (Россия)"
```

```suggestion
				"ru": "Русский"
```
Надо бы ещё и файл в `ru.json` переименовать